tests/users/test_data.py

```diff
@@ -174,3 +174,15 @@ async def test_find_or_create_b2c_user(
         assert int(user.handle.split("-")[-1])
 
     assert user == snapshot(matcher=path_type({"handle": (str,)}))
+
+
+async def test_check_is_empty(mongo, users_data, fake2, authorization_client, pg):
```

Comment:
Needs doc string. What is the test doing.

Please don't combine test cases. Either use `pytest.mark.parametrize` or write two separate tests.